app/views/layouts/_body.html.haml

```diff
@@ -2,6 +2,8 @@
 
 .ui.pusher
   .ui.main.container
+    = render partial: "layouts/service_announcements", locals: { service_announcements: @active_service_announcements }
```

Comment:
I'd ditch the local here and just use the helper method

Reply:
Something like?
```
﻿module ApplicationHelper
  def render_service_announcements
    render partial: "layouts/service_announcements", locals: { service_announcements: ServiceAnnouncements.displayable }
  end
end
```

Reply:
I'm talking about just calling the `helper_method :active_service_announcements` defined up in ApplicationController

Reply:
If I do something like this though, I don't need to set an instance variable for the active announcements (only need to render the list once per page) and this long line gets pulled out of the templates into a nicer little method call.